java/pom.xml

```diff
@@ -17,7 +17,7 @@
         <module>runtime-dev</module>
         <module>cli</module>
         <module>test</module>
-        <module>example</module>
```

Comment:
test_cluster.sh is using the example jar for cluster mode testing in single box. May you also update that script since you're renaming the jar?